src/utils/utf8.c

```diff
@@ -25,15 +25,30 @@
 #include <unicode/utypes.h>
 #include <unicode/ucasemap.h>
 #include <unicode/unorm2.h>
+#include <unicode/unistr.h>
+#include <unicode/utrans.h>
 
 #include "strbuf.h"
 #include "utf8.h"
 #include "utils.h"
 
+/*
+* UCI Rule for transliteration.
+* see more: https://unicode-org.github.io/icu/userguide/transforms/general/
+*/
+static const char NFKD_RULE[] = "NFKD; [:Nonspacing Mark:] Remove; Latin-ASCII; NFKC";
+
+/*
+* Max string buffer
+*/
+static const int MAX_BUFFER_SIZE = 128;
```

Comment:
This might be too short for some long words (perhaps in German). I am going to replace it (see the FIXME marks) with a dynamic buffer. Anyway, don't worry about it, but just FYI.